### PR TITLE
[ADD] pybrake new framework support: Pycnic

### PR DESCRIPTION
**Pull request by smurf-U**

### CODE REVIEW

File: examples/pycnic/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    curl "http://localhost:3000/weather/<austin/pune/santabarbara>"
+    ```
+
+   The below curl command will raise `404 Not Found` error.
+
+    ```bash
+    curl -I "http://localhost:3000/weather"
+    ```
+
+   The below curl command will raise `500 Internal server error` error.
```
Comment (kyrylo) [inline examples/pycnic/README.md:61]:
```suggestion
   The below curl command will raise `500 Internal Server Error` error.
```

File: examples/pycnic/main.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                'https://airbrake.github.io/weatherapi/weather/' +
+                location_name) as f:
+            return f.json()
+
+
+class App(PybrakeEnableWSGI):
+    config = {
+        'PYBRAKE': dict(
+            project_id=445187,
+            project_key='52f09978622bea0ec9cd8015e1eba322',
```
Comment (kyrylo) [inline examples/pycnic/main.py:75]: Did you mean to leave the credentials in the code?

Comment (smurf-U) [reply examples/pycnic/main.py:75]: :open_mouth: my bad.

**Comment by smurf-U** [comment]
PTAL

**Review by kyrylo — APPROVED** [review]

**Pull request merged**